contrib/test/integration/build/cri-o.yml

```diff
@@ -78,7 +78,8 @@
     dest: /etc/crio/crio.conf.d/01-overlay.conf
     content: |
       [crio]
-      storage_driver = "overlay"
+      default_storage = "overlay"
+      storage_driver = [ "overlay" ]
```

Comment:
Do we wanna make the change backwards compatible, like keeping `storage_driver` and add something like `additional_storage_drivers`?

Reply:
yes, this is important! I can leave `storage_driver` and `storage_opt`  and use this as the default and add what you proposed as a table:
```toml
[crio]
storage_driver = "overlay"
storage_opt = ["overlay.mountopt=nodev,metacopy=on","overlay.mountopt=nodev,metacopy=on"]

[crio.additional_storage.devicemapper]
options = ["dm.directlvm_device=/dev/loop0", "dm.fs=ext4", "dm.directlvm_device_force=true"]
[crio.additional_storage.btrfs]


```
Same for the crio cli flags.